openstack/content-repo/bin/sleep-until-midnight

```diff
@@ -9,19 +9,33 @@ function sleepuntil ()
 
   sleep_seconds=$(( $target_epoch - $current_epoch ))
   #If time is already reached, wait for the next day
-  if [ ${sleep_seconds} -lt 0 ]; then
-    sleep_seconds=$((86400 + sleep_seconds))
+  if [ $sleep_seconds -lt 0 ]; then
+    sleep_seconds=$(( 86400 + $sleep_seconds ))
```

Comment:
No `$` needed here before `sleep_seconds`. I think it will work, too, but it's not recommended. (Same applies the other `$((...))` instances below.)